pkg/scaffold/controller/controller.go

```diff
@@ -120,8 +120,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 	{{ .Resource.Group}}{{ .Resource.Version }} "{{ .ResourcePackage }}/{{ .Resource.Group}}/{{ .Resource.Version }}"
+)
+
+var log = logf.Log.WithName("controller")
```

Comment:
Maybe the name should be `{{ lower .Resource.Kind }}-controller` instead?

Reply:
I think this is probably a good idea to distinguish controllers in log.
You are welcome to send us a PR for this enhancement.

Reply:
Filed https://github.com/kubernetes-sigs/kubebuilder/pull/609

Reply:
Adding a res kind is better.
Thanks @mengqiy @janetkuo